go tests failing because change of defination of function in latest bcc master branch

bcc commit "Support for hardware offload" d147588ebe35b7cd2b4d253a7da18bef253ea78d

added extra argument "const char *dev_name" which is failing this test.
 
```
# go test -tags integration -v ./...
# github.com/iovisor/gobpf/bcc
  ../github.com/iovisor/gobpf/bcc/module.go:98:40: not enough arguments in call to _Cfunc_bpf_module_create_c_from_string
	have (*_Ctype_char, number, **_Ctype_char, _Ctype_int, _Ctype__Bool)
	want (*_Ctype_char, _Ctype_uint, **_Ctype_char, _Ctype_int, _Ctype__Bool, *_Ctype_char)
FAIL	gobpf [build failed]
# gobpf/bcc
bcc/module.go:98:40: not enough arguments in call to _Cfunc_bpf_module_create_c_from_string
	have (*_Ctype_char, number, **_Ctype_char, _Ctype_int, _Ctype__Bool)
	want (*_Ctype_char, _Ctype_uint, **_Ctype_char, _Ctype_int, _Ctype__Bool, *_Ctype_char)
```

I came across the same problem, do you have patch to fix it? still new to golang, not sure how to fix it myself
<pre>
go test -tags integration -v ./... github.com/iovisor/gobpf/bcc
/usr/local/go/src/github.com/iovisor/gobpf/bcc/module.go:98:8: error: not enough arguments
  c := C.bpf_module_create_c_from_string(cs, 2, (**C.char)(&cflagsC[0]), C.int(len(cflagsC)), (C.bool)(true))
        ^
FAIL	_/usr/src/gobpf [build failed]
# _/usr/src/gobpf/bcc
bcc/module.go:98:8: error: not enough arguments
  c := C.bpf_module_create_c_from_string(cs, 2, (**C.char)(&cflagsC[0]), C.int(len(cflagsC)), (C.bool)(true))
        ^
</pre>

how to add the "const char *dev_name" in C.bpf_module_create_c_from_string?

--- a/bcc/module.go
+++ b/bcc/module.go
@@ -95,7 +95,7 @@ func newModule(code string, cflags []string) *Module {
 	}
 	cs := C.CString(code)
 	defer C.free(unsafe.Pointer(cs))
-	c := C.bpf_module_create_c_from_string(cs, 2, (**C.char)(&cflagsC[0]), C.int(len(cflagsC)), (C.bool)(true))
+	c := C.bpf_module_create_c_from_string(cs, 2, (**C.char)(&cflagsC[0]), C.int(len(cflagsC)), (C.bool)(true), nil)
 	if c == nil {
 		return nil
 	}

